.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
Consider adding a comment or explanation for the `.venv/` entry to clarify that it is used to ignore virtual environment directories. This can help other developers understand the purpose of this entry.